tuf/client_rework/updater_rework.py

```diff
@@ -516,7 +331,7 @@ def _preorder_depth_first_walk(self, target_filepath) -> Dict:
         target = None
         role_names = [("targets", "root")]
         visited_role_names = set()
-        number_of_delegations = settings.MAX_NUMBER_OF_DELEGATIONS
+        number_of_delegations = MAX_DELEGATIONS
 
         # Ensure the client has the most up-to-date version of 'targets.json'.
         # Raise 'exceptions.NoWorkingMirrorError' if the changed metadata
```

Comment:
This is not actually the case: NoWorkingMirrors don't happen anymore, there's TODOs about download errors